doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -296,35 +296,35 @@ The requirements in this section are to be delivered in addition to those in [se
 | `req.inf.ntw.13` | Infrastructure | Network | The platform must allow specifying multiple separate IP pools. Tenants are required to select at least one IP pool that is different from the control infrastructure IP pool or other tenant IP pools. | |
-| `req.inf.ntw.14` | Infrastructure | Network | The platform must allow NATless traffic (i.e. exposing the pod IP address directly to the outside), allowing source and destination IP addresses to be preserved in the traffic headers from workloads to external networks. This is needed e.g. for signaling applications, using SIP and Diameter protocols.| |
-| `req.inf.vir.01` | Infrastructure | Virtual Infrastructure | The Architecture must support the capability for Containers to consume infrastructure resources abstracted by Host Operating Systems that are running within a virtual machine. | |
+| `req.inf.ntw.14` | Infrastructure | Network | The platform must allow NATless traffic (i.e. exposing the pod IP address directly to the outside), allowing source and destination IP addresses to be preserved in the traffic headers from workloads to external networks. This is needed e.g. for signaling applications, using SIP and Diameter protocols.|[ra2.ntw.011](chapter04.md#45-networking-solutions)|
+| `req.inf.vir.01` | Infrastructure | Virtual Infrastructure | The Architecture must support the capability for Containers to consume infrastructure resources abstracted by Host Operating Systems that are running within a virtual machine. |[ra2.ch.005](chapter04.md#42-kubernetes-node)<br>[ra2.ch.011](chapter04.md#42-kubernetes-node)|
 | `req.inf.phy.01`  | Infrastructure |  Physical Infrastructure | The Architecture must support the capability for Containers to consume infrastructure resources abstracted by Host Operating Systems that are running within a physical server. | |
 | `req.kcm.gen.01` | Kubernetes Cluster | General | The Architecture **must** support policy driven horizontal auto-scaling of Kubernetes cluster. | |
-| `req.kcm.gen.02` | Kubernetes Cluster | General | The Architecture **must** enable workload resiliency. | |
-| `req.int.api.01` | API | General | The Architecture **must** leverage the Kubernetes APIs to discover and declaratively manage compute (virtual and bare metal resources), network, and storage. | |
-| `req.int.api.02` | API | General | The Architecture **must** support the usage of a Kubernetes Application package manager using the Kubernetes API, like Helm v3. | |
+| `req.kcm.gen.02` | Kubernetes Cluster | General | The Architecture **must** enable workload resiliency. |[ra2.k8s.004](chapter04.md#43-kubernetes)|
+| `req.int.api.01` | API | General | The Architecture **must** leverage the Kubernetes APIs to discover and declaratively manage compute (virtual and bare metal resources), network, and storage. |[ra2.ntw.001](chapter04.md#45-networking-solutions)<br>[ra2.ntw.008](chapter04.md#45-networking-solutions)<br>[ra2.app.006](chapter04.md#49-kubernetes-workloads)|
```

Comment:
Why ra2.ntw.008 - SR-IOV?
also we need for both compoute and storage -- CRI, CSI?

Reply:
ra2.ntw.008 is about Device Plugin, which is about leveraging k8s APIs (DPAPI being one). Less about the SR-IOV part.

Reply:
@tomkivlin Sorry wasn't clear. 
| `req.int.api.01` | API | General | The Architecture **must** leverage the Kubernetes APIs to discover and declaratively manage compute (virtual and bare metal resources), network, and storage.
-- | -- | --|-- 

The compute and storage are not addressed; only network is.


ra2.ntw.008 | SR-IOV Device Plugin for Network Intensive | When hosting workloads that match the Network Intensive profile and  require SR-IOV acceleration, a Device Plugin for SR-IOV must be used to  configure the SR-IOV devices and advertise them to the kubelet.
-- | -- | --

Reply:
Oh I see, yes agreed. I've added a clarification in https://github.com/cntt-n/CNTT/pull/2204/commits/9c7c228c3a70e396d17293a46f470804172e75b0.